crates/masonry/src/widget/textbox.rs

```diff
@@ -249,8 +254,7 @@ impl Widget for Textbox {
         let max_advance = if self.line_break_mode != LineBreaking::WordWrap {
             None
         } else if bc.max().width.is_finite() {
-            // TODO: Does Prose have different needs here?
-            Some(bc.max().width as f32 - 2. * TEXTBOX_PADDING as f32)
+            Some(bc.max().width as f32 - 2. * TEXTBOX_PADDING as f32 - 2. * TEXTBOX_MARGIN as f32)
```

Comment:
Why not just do the math at 64 bit and then one cast to `f32`?

`Some((bc.max().width - 2. * TEXTBOX_PADDING - 2. * TEXTBOX_MARGIN) as f32)`